src/components/PersonLink.tsx

```diff
@@ -0,0 +1,59 @@
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+const PersonLink: React.FC<Props> = ({ person, people }) => {
+  const params = useParams();
+  const { sex, name, born, died, fatherName, motherName, slug } = person;
+
+  const mother = motherName ? people.find(p => p.name === motherName) : null;
+
+  const father = fatherName ? people.find(p => p.name === fatherName) : null;
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn({ 'has-background-warning': params.slug === slug })}
+    >
+      <td>
+        <Link
+          className={cn({ 'has-text-danger': sex === 'f' })}
+          to={`/people/${slug}`}
+        >
+          {name}
+        </Link>
+      </td>
+
+      <td>{sex}</td>
+      <td>{born}</td>
+      <td>{died}</td>
+      <td>
+        {mother ? (
+          <Link className="has-text-danger" to={`/people/${mother.slug}`}>
+            {mother.name}
+          </Link>
+        ) : motherName ? (
+          motherName
+        ) : (
+          '-'
+        )}
+      </td>
+      <td>
+        {father ? (
+          <Link to={`/people/${father.slug}`}>{father.name}</Link>
+        ) : fatherName ? (
+          fatherName
+        ) : (
+          '-'
```

Comment:
Similar to the comment for the mother's link, ensure consistency in how you handle missing information for the father. If `fatherName` is `null` or an empty string, the placeholder '-' should be displayed.